### ISSUE TITLE
Turn off Spiral View

### ISSUE THREAD

**Issue by harishchopra86** [venits/react-native-map-clustering#132]
Can I turn off Spiral View of pins on same location so that it shows cluster with count?

**Comment by nashashibi** [comment]
I don't think you there's an out-of-the-box way to do it. You need to either fork it or edit the pacakge in your node_modules (I did the latter).

Go to **ClusteredMapView.js** 
1. find this line: `let positions = generateSpiral(...etc` and comment/remove it. 
2. replace `updateSpiderMarker(positions);` with:
```
updateSpiderMarker([{
  latitude: markers[0].geometry.coordinates[1],
  longitude: markers[0].geometry.coordinates[0],
  image: clusterChildren[0].properties.image,
  onPress: clusterChildren[0].properties.onPress
}]);
```

### IMPORTANT!
the above solution will only work if you provide `image` prop to your markers in your code.

**Comment by venits** [comment]
I released a new version of this module: `3.2.0`. 
I added new prop **spiralEnabled**.

You simply add this to MapView to disable spiral view:
```js
spiralEnabled={false}
```
